ballerina/sub_listener.bal

```diff
@@ -15,10 +15,12 @@
 // under the License.
 
 import ballerina/http;
+import ballerina/lang.runtime;
 import ballerina/log;
 
 # Represents a Subscriber Service listener endpoint.
 public class Listener {
+    private final decimal gracefulStopTime = 30;
```

Comment:
Let's change this gracefulStopTime to `gracefulShutdownPeriod`

Reply:
Also, lets make it configurable.

Reply:
Fixed with [110266fa0e834b6319b9f2f882acc2da473cec1a](https://github.com/ballerina-platform/module-ballerina-websub/pull/363/commits/110266fa0e834b6319b9f2f882acc2da473cec1a).